### ISSUE TITLE
Installation without composer

### ISSUE THREAD

**Issue by taalas** [portrino/typo3-whoops#1]
Is it possible to install this package without composer?

**Comment by aWuttig** [comment]
No. It is no TYPO3 Extension. You can try to put the code within your typo3conf and try to autoload the classes. But a quick installation via EM is not possible.

**Comment by mbrodala** [comment]
It would be possible to [bundle `filp/whoops` for the Classic Mode](http://insight.helhum.io/post/148112375750/how-to-use-php-libraries-in-legacy-extensions) however. In any case you should probably contact the author of the [`whoops` extension](https://extensions.typo3.org/extension/whoops/) and ask for a transfer of the extension key.

**Comment by aWuttig** [comment]
I think we will not support classic mode, because we use the mentioned filp/whoops php package via composer.
